ctapipe/atmosphere/__init__.py

```diff
@@ -0,0 +1 @@
+from .model import *
```

Comment:
Start imports and no `__all__` mess with sphinx, please import explicitly and add an `__all__`